apps/meteor/server/settings/e2e.ts

```diff
@@ -16,6 +16,14 @@ export const createE2ESettings = () =>
 			enableQuery: { _id: 'E2E_Enable', value: true },
 		});
 
+		await this.add('E2E_Enable_Encrypt_Files', true, {
+			type: 'boolean',
+			i18nLabel: 'E2E_Enable_Encrypt_Files',
+			i18nDescription: 'E2E_Enable_Encrypt_Files_Description',
```

Comment:
```suggestion
```
It should be ok to omit this

Reply:
I think it should be okay to mention these properties, good to be explicit, especially for people who are new to this project and codebase.